packages/website/components/SandpackRenderer.tsx

```diff
@@ -50,11 +50,13 @@ export function SandpackRenderer(props: {
           react: '^17.0.0',
           'react-dom': '^17.0.0',
           'react-scripts': '^4.0.0',
+          '@contentful/f36-forms': 'latest',
```

Comment:
All components you need should be available in `@contentful/f36-components` so I don't see a reason to add `forms` package here